run-container.sh

```diff
@@ -1,37 +1,12 @@
 #!/usr/bin/env bash
 
-# Requires
-# - kubeconfig file mounted to /usr/tnf/kubeconfig/config
-#   If more than one kubeconfig needs to be used, bind
-#   additional volumes for each kubeconfig, e.g.
-#     - /usr/tnf/kubeconfig/config
-#     - /usr/tnf/kubeconfig/config.2
-#     - /usr/tnf/kubeconfig/config.3
-# - TNF config files mounted into /usr/tnf/config
-# - A directory to output claim into mounted at /usr/tnf/claim
-# - A $KUBECONFIG environment variable passed to the TNF container
-#   containing all paths to kubeconfigs located in the container, e.g.
-#   KUBECONFIG=/usr/tnf/kubeconfig/config:/usr/tnf/kubeconfig/config.2
-
-REQUIRED_VARS=('LOCAL_KUBECONFIG' 'LOCAL_TNF_CONFIG' 'OUTPUT_LOC')
-REQUIRED_VARS_ERROR_MESSAGES=(
-	'KUBECONFIG is invalid or not given. Use the -k option to provide path to one or more kubeconfig files.'
-	'TNFCONFIG is required. Use the -t option to specify the directory containing the TNF configuration files.'
-	'OUTPUT_LOC is required. Use the -o option to specify the output location for the test results.'
-)
-
-TNF_IMAGE_NAME=test-network-function
-TNF_IMAGE_TAG=latest
-TNF_OFFICIAL_ORG=quay.io/testnetworkfunction/
-TNF_OFFICIAL_IMAGE="${TNF_OFFICIAL_ORG}${TNF_IMAGE_NAME}:${TNF_IMAGE_TAG}"
-
 CONTAINER_TNF_DIR=/usr/tnf
 CONTAINER_TNF_KUBECONFIG_FILE_BASE_PATH="$CONTAINER_TNF_DIR/kubeconfig/config"
 CONTAINER_DEFAULT_NETWORK_MODE=bridge
 
 usage() {
 	read -d '' usage_prompt <<- EOF
-	Usage: $0 -t TNFCONFIG -o OUTPUT_LOC [-i IMAGE] [-k KUBECONFIG] [-n NETWORK_MODE] SUITE [... SUITE]
+	Usage: $0 -t TNFCONFIG -o OUTPUT_LOC [-i IMAGE] [-k KUBECONFIG] [-n NETWORK_MODE] [-d DNS_RESOLVER_ADDRESS] SUITE [... SUITE]
```

Comment:
Looking at the `run-cnf-suites.sh` file, the `usage()` helper function should probably be moved from `run-container.sh` to the newly created `run-tnf-container.sh` file.

The usage prompt uses `$0` to display the name of the script (`run-container.sh`).
This means that the usage prompt displays incorrect usage info and examples, e.q.:
```
Usage: ./run-container.sh -t TNFCONFIG -o OUTPUT_LOC [-i IMAGE] [-k KUBECONFIG] [-n NETWORK_MODE] [-d DNS_RESOLVER_ADDRESS] SUITE [... SUITE]

[...]

Examples
  ./run-container.sh -t ~/tnf/config -o ~/tnf/output diagnostic generic

  Because -k is omitted, run-container.sh will first try to autodiscover local kubeconfig files.
  If it succeeds, the diagnostic and generic tests will be run using the autodiscovered configuration.
  The test results will be saved to the '~/tnf/output' directory on the host.
```
Examples provided in the usage message no longer work, as one has to call `./run-tnf-container.sh` instead.

Reply:
Will move the usage function